annotations/build.gradle.kts

```diff
@@ -12,6 +12,8 @@ java { sourceCompatibility = JavaVersion.VERSION_21 }
 
 repositories { mavenCentral() }
 
+dependencyManagement { imports { mavenBom("org.testcontainers:testcontainers-bom:1.19.7") } }
```

Comment:
With this configuration, you can actually import the Maven POM, using the SB dependency management plugin.